src/translations/translation.types.d.ts

```diff
@@ -0,0 +1,14 @@
+import { translations as enTranslations } from './translations.en.js';
+import { translations as frTranslations } from './translations.fr.js';
+
+type Translations = Omit<typeof frTranslations & typeof enTranslations, 'LANGUAGE'>;
+type TranslationKeys = keyof Translations;
+type Function = (...args: any) => any;
```

Comment:
**suggestion:** I didn't think of it when we create that but since `Function` actually exists, maybe we should go another name?

```suggestion
type StandardArrowFunction = (...args: any) => any;
```

```suggestion
type ArrowFunctionWithArgs = (...args: any) => any;
```

(none of these names convince me, I'll try to think of something better...)